runtime/contrib/android/README.md

```diff
@@ -0,0 +1,44 @@
+## How to build
+
+after building `onert`
```

Comment:
@YongseopKim How about mention that library files in `Product/out/lib` will be used?
Or changing symbolic link for library files from  `Product/out/lib` to `Product/aarch64-android.release/out/lib` would be better.

Reply:
This seems not easy. I'll fix it on next pr.